packages/test-utils/src/transforms/babelTransform.cjs

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable */
```

Comment:
These transforms are from Create React App. Adding eslint-disable so ESLint stops asking to change them.